runtime-common/stdlib/string/string-functions.h

```diff
@@ -18,7 +19,7 @@ string f$addslashes(const string &str) noexcept;
 
 string f$hex2bin(const string &str) noexcept;
 
-inline string f$bin2hex(const string &str) noexcept {
+inline string bin2hex_impl(std::string_view str) noexcept {
```

Comment:
Let's add it namespace like `string_functions_impl_`

Reply:
Fixed